cypress/e2e/room/room-header.spec.ts

```diff
@@ -190,4 +192,103 @@ describe("Room Header", () => {
+                    Hello World
+                </body>
+            </html>
+        `;
+
+        let widgetUrl: string;
+        let roomId: string;
+
+        beforeEach(() => {
+            cy.serveHtmlFile(WIDGET_HTML).then((url) => {
+                widgetUrl = url;
+            });
+
+            cy.createRoom({ name: ROOM_NAME }).then((id) => {
+                roomId = id;
+
+                // setup widget via state event
+                cy.getClient()
+                    .then(async (matrixClient) => {
+                        const content: IWidget = {
+                            id: WIDGET_ID,
+                            creatorUserId: "somebody",
+                            type: "widget",
+                            name: "widget",
+                            url: widgetUrl,
+                        };
+                        await matrixClient.sendStateEvent(roomId, "im.vector.modular.widgets", content, WIDGET_ID);
+                    })
+                    .as("widgetEventSent");
+
+                // set initial layout
+                cy.getClient()
+                    .then(async (matrixClient) => {
+                        const content = {
+                            widgets: {
+                                [WIDGET_ID]: {
+                                    container: "top",
+                                    index: 1,
+                                    width: 100,
+                                    height: 0,
+                                },
+                            },
+                        };
+                        await matrixClient.sendStateEvent(roomId, "io.element.widgets.layout", content, "");
+                    })
+                    .as("layoutEventSent");
+            });
+
+            cy.all([cy.get<string>("@widgetEventSent"), cy.get<string>("@layoutEventSent")]).then(() => {
+                // open the room
+                cy.viewRoomByName(ROOM_NAME);
+            });
+        });
+
+        it("should highlight the apps button", () => {
+            // Assert that AppsDrawer is rendered
+            cy.get(".mx_AppsDrawer").should("exist");
+
+            cy.get(".mx_RoomHeader").within(() => {
+                cy.findByRole("button", { name: "Hide Widgets" })
+                    .should("exist") // Assert that the apps button is rendered
+                    .then(($btn) => {
+                        // Note it is not possible to get CSS values of a pseudo class with "have.css".
+                        const color = $btn[0].ownerDocument.defaultView // get window reference from element
+                            .getComputedStyle($btn[0], "before") // get the pseudo selector
+                            .getPropertyValue("background-color"); // get "background-color" value
+
+                        // Assert the value is equal to $accent == hex #0dbd8b == rgba(13, 189, 139)
+                        expect(color).to.eq("rgb(13, 189, 139)");
```

Comment:
Can we check for a CSS class instead of an exact colour? I wouldn't want our test to fail if we tweak the colours.

Even better would be to check for an aria-label that said it was selected, or something like that, but it might not exist...

Reply:
> aria-label that said it was selected

Aria labels shouldn't be used for that, `aria-selected`, `aria-checked`, `aria-pressed` etc should be used instead

Reply:
Ah great, thank you.

Reply:
With https://github.com/matrix-org/matrix-react-sdk/pull/10934/commits/cd4159458fadca1ebccf5366b1a1ae98e683685a I replaced assertions related to `color` with `aria-selected`, hope this should be fine.